src/Autosuggest.js

```diff
@@ -156,6 +159,14 @@ class Autosuggest extends Component {
     }
   }
 
+  maybeSelectFirstSuggestion() {
+    const { selectFirstSuggestion, multiSection, updateFocusedSuggestion } = this.props;
+
+    if (selectFirstSuggestion) {
+      updateFocusedSuggestion(multiSection ? 0 : null, 0);
+    }
+  }
```

Comment:
Thinking more about this, I feel like it doesn't worth the effort. Interested to hear what others think.